cmd/tether/main_linux.go

```diff
@@ -123,14 +123,22 @@ func halt() {
 
 func startSignalHandler() {
 	sigs := make(chan os.Signal, 1)
-	signal.Notify(sigs, syscall.SIGHUP)
+	signal.Notify(sigs, syscall.SIGHUP, syscall.SIGUSR1, syscall.SIGUSR2, syscall.SIGPWR, syscall.SIGTERM, syscall.SIGINT)
 
 	go func() {
 		for s := range sigs {
 			switch s {
 			case syscall.SIGHUP:
 				log.Infof("Reloading tether configuration")
 				tthr.Reload()
+			case syscall.SIGUSR1, syscall.SIGUSR2, syscall.SIGPWR:
```

Comment:
Why include _both_ `SIGUSR1` and `SIGUSR2` here? Is there a chance we'd ever want to use one for something other than "stop" in the future?